Website/plugins/page-styling/page-styling.raku

```diff
@@ -107,23 +105,26 @@ use v6.d;
         qq:to/BLOCK/
           <div id="navMenu" class="navbar-menu">
             <div class="navbar-start">
-                <a class="navbar-item" href="/language">
-                    Language
+                <a class="navbar-item" href="/introduction" title="Getting started, Tutorials, Migration guides">
+                    Introduction
                 </a>
-                <a class="navbar-item" href="/types">
+                <a class="navbar-item" href="/reference" title="Fundamentals, General reference">
+                    Reference
+                </a>
+                <a class="navbar-item" href="/miscellaneous" title="Programs, Experimental">
+                    Miscellaneous
+                </a>
+                <a class="navbar-item" href="/types" title="The core types (classes) available">
                     Types
                 </a>
-                <a class="navbar-item" href="/routines">
+                <a class="navbar-item" href="/routines" title="Searchable table of routines">
                     Routines
                 </a>
-                <a class="navbar-item" href="/programs">
-                    Programs
-                </a>
-                <a class="navbar-item" href="https://raku.org">
-                    Raku™ Homepage
+                <a class="navbar-item" href="https://raku.org" title="Home page for community">
+                    Raku™
                 </a>
-                <a class="navbar-item" href="https://kiwiirc.com/client/irc.libera.chat/#raku">
-                    Chat with us
+                <a class="navbar-item" href="https://kiwiirc.com/client/irc.libera.chat/#raku" title="IRC live logs">
+                    Chat
```

Comment:
these aren't the logs, it's an in-browser IRC Client

Reply:
@coke Do you have a better 'title' text that is more accurate?

Reply:
s/logs/chat/ is fine.